ci/docker/bosh-oracle-cpi-boshrelease/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:15.04
+MAINTAINER CF Dev <cloud_dev_cf_ww_grp@oracle.com>
```

Comment:
Should replace with an external public email address.

Reply:
Yes. Do we have one?

Reply:
I was thinking it would your publicly visible address used at github, but maybe not.  @brunoborges do you have a suggestion here?